Utilities/Analyzer.cc

```diff
@@ -45,8 +37,6 @@ class Analyzer{
     else EntryMax = r->GetEntries();
     cout << "Processing " << EntryMax << " events" << endl;
     progress = new Progress(EntryMax, conf->ProgressInterval);
-    if (!IsMC) datasel = new DataSelection(conf);
```

Comment:
DataSelection is not done by the RegionAssociation. It is not object selection. Instead, it considers whether a run/luminosity block is a valid one in data with reading golden JSON files.
This step probably is covered by skims, but I think a double check is fine.

Reply:
Okay, I mixed that up. The object selection cuts you had were done by the RegionAssociation. The luminosity block selection should have been done when skimming the data. You can crosscheck whether this has been done or not, but we shouldn't do it all over again. That may cost little runtime, but it's another source of potential inconsistency, if different jsons are chosen, because no matter what you do then, the outcome will be wrong, since you can't restore already skimmed events at this step.